lib/gringotts/gateways/cams.ex

```diff
@@ -323,10 +346,20 @@ defmodule Gringotts.Gateways.Cams do
     defp set_success(opts, %{"response_code" => response_code}) do
       opts ++ [success: response_code == "100"]
```

Comment:
What is happening here? Assignment or comaprison. It is confusing.

Reply:
its comparison. It will set the success true or false.